api_app/script_analyzers/observable_analyzers/firehol_iplist.py

```diff
@@ -0,0 +1,81 @@
+import os
+import logging
+import re
+import traceback
+import requests
+import ipaddress
+
+from api_app.exceptions import AnalyzerRunException
+from api_app.script_analyzers import classes
+from intel_owl import settings
+
+logger = logging.getLogger(__name__)
+
+db_name = "firehol_level1.txt"
+database_location = f"{settings.MEDIA_ROOT}/{db_name}"
+
+
+class FireHol_IPList(classes.ObservableAnalyzer):
+    def run(self):
+        result = {"found": False}
+
+        if not os.path.isfile(database_location):
+            self.updater()
+
+        with open(database_location, "r") as f:
+            db = f.read()
+
+        if self.observable_name in db:
+            result["found"] = True
```

Comment:
we cannot use a boolean for this. This is not maintanable considering that your PR does not solve the issue entirely.

This PR can be accepted only in the case that you already prepare this analyzer to have the same output schema that it would have once the issue is solved entirely. This means that you should provide the name of the lists + the category of the lists where the search IP address matched. In this case it would at maximum show the one that you integrated

Reply:
For simplicity, we can make it: `result[list_name] = True/False`.

Add `list_name` parameter to the `additional_config_parameters` inside the analyzer config.

Reply:
Yes, that makes sense!